lib/ecstatic.js

```diff
@@ -57,27 +57,39 @@ var ecstatic = module.exports = function (dir, options) {
     catch (err) {
       return status[400](res, next, { error: err });
     }
-
-    var file = path.normalize(
-          path.join(root,
-            path.relative(
-              path.join('/', baseDir),
-              pathname
+    
+    // try and serve assets
+    var assetsDirectory = '_ecstatic-assets'
```

Comment:
Do you think those lines would look good along with the http style status pages in https://github.com/jfhbrook/node-ecstatic/pull/151/files ? I might try to copypasta this a little.